### ISSUE TITLE
bug: data fetch fails from invalid api url

### ISSUE THREAD

**Issue by onordbo** [franzwilhelm/raycast-toggl-track#1]
[The Toggl API](https://developers.track.toggl.com/docs/api/authentication) endpoint/url changed from v8 to v9, breaking the extension connection to the connected Toggl account.

**Comment by franzwilhelm** [comment]
True, but I use v8 in this repo, so shouldn't cause any problems unless I migrate to v9? The extension seems to be working as before. Could you provide me with details on how to reproduce?

**Comment by onordbo** [comment]
Huh, that's strange. It broke out of the box for me when I installed the plugin from the Raycast store 🤔 Maybe something else is wrong on my end, although it looks like I'm properly connected to the internet, and everything else is fetching fine as far as I can see.

1. Fresh install (from the [store](https://www.raycast.com/franzwilhelm/toggl-track))
2. Dropped in my api-token
3. Ran the track time command
4. Got the message "Failed to load data from Toggle. Please try again." _(see ss)_

<img width="886" alt="image" src="https://user-images.githubusercontent.com/4263132/187166297-33871d20-2ed5-43a7-80e3-f23edcacce4b.png">
